modules/tensor_mechanics/src/materials/ComputeFiniteStrain.C

```diff
@@ -11,6 +11,7 @@ InputParameters validParams<ComputeFiniteStrain>()
 {
   InputParameters params = validParams<ComputeStrainBase>();
   params.addClassDescription("Compute a strain increment and rotation increment for finite strains.");
+  params.set<bool>("stateful_displacements") = true;
```

Comment:
I'd prefer if you don't use this syntax even though it seems like the most correct one. The problem is copy 'n' paste... People tend to start bypassing the "add" methods when they see this. I like just re-adding the parameter again and setting it's value.